com.github.you_win.VPupPr.appdata.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
+  <id>com.github.youwin.VPupPr</id>
+  <launchable type="desktop-id">com.github.youwin.VPupPr.desktop</launchable>
+  <name>Virtual Puppet Project</name>
+  <summary>VTuber application made with Godot 3.4</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT</project_license>
+  <releases>
+    <release>type="stable" version="0.8.1"</release>
```

Comment:
```suggestion
    <release type="stable" version="0.8.1"></release>
```

Reply:
There we go. I've added the github repo using the tool you linked (thanks for that, btw!) and it added all of the releases automatically.